modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -75,13 +77,15 @@ struct GAPI_Streaming: public ::testing::TestWithParam<KernelPackage> {
         {
         case KernelPackage::OCV:
             return cv::gapi::combine(core::cpu::kernels(),
-                                     imgproc::cpu::kernels());
+                                     imgproc::cpu::kernels(),
+                                     streaming::kernels());
             break;
 
         case KernelPackage::OCV_FLUID:
             return cv::gapi::combine(core::cpu::kernels(),
                                      imgproc::cpu::kernels(),
-                                     core::fluid::kernels());
+                                     core::fluid::kernels(),
+                                     streaming::kernels());
```

Comment:
...so most of these changes should be reverted in this file

Reply:
There is use_only in these tests to make sure correct kernels are launched